pkg/providers/launchtemplate/launchtemplate.go

```diff
@@ -424,3 +431,26 @@ func (p *Provider) DeleteLaunchTemplates(ctx context.Context, nodeClass *v1beta1
 	}
 	return nil
 }
+
+func (p *Provider) ResolveClusterCIDR(ctx context.Context) error {
+	if p.ClusterCIDR != nil {
+		return nil
+	}
+	out, err := p.eksapi.DescribeClusterWithContext(ctx, &eks.DescribeClusterInput{
+		Name: aws.String(options.FromContext(ctx).ClusterName),
+	})
+	if err != nil {
+		return fmt.Errorf("failed to resolve cluster CIDR, %w", err)
```

Comment:
```suggestion
		return fmt.Errorf("resolving cluster CIDR, %w", err)
```

nit: we prefer the gerund as a style thing for our errors